stack/create.go

```diff
@@ -36,6 +36,10 @@ const (
 
 	// ErrStackAlreadyExists indicates that the stack already exists and cant be created.
 	ErrStackAlreadyExists errors.Kind = "stack already exists"
+
+	// ErrStackDefaultCfgFound indicates that the dir already has a default
+	// stack configuration.
+	ErrStackDefaultCfgFound errors.Kind = "default cfg file already exists"
```

Comment:
```suggestion
	ErrStackDefaultCfgFound errors.Kind = "default configuration file for stack already exists"
```
not sure if better, just trying to make it clear that this is the default file we use for stacks...